data_model/src/query.rs

```diff
@@ -75,6 +75,8 @@ pub enum QueryBox {
     FindAssetQuantityById(FindAssetQuantityById),
     /// [`FindTotalAssetQuantityByAssetDefinitionId`] variant.
     FindTotalAssetQuantityByAssetDefinitionId(FindTotalAssetQuantityByAssetDefinitionId),
+    /// [`IsAssetCreator`] variant.
+    IsAssetCreator(IsAssetCreator),
```

Comment:
this arguably doesn't bring much? user can query for the asset definition and check the creator?

Reply:
Hm, I didn't think about querying asset definition... But I thought that it will be a popular query and not only for validators.

But okay, I see, there are you and Aleksandr on your side, so I will remove this query

Reply:
Well, it turns out that simple `AssetDefintion` does not contain the owner id. It's stored in WSV and called `AssetDefinitionEntry`.

Introducing this query was the simpliest way, I believe

Reply:
should our queries return `AssetDefinitionEntry` instead of `AssetDefinition`?

Reply:
Let's move this discussion into #3252.

This PR is already pretty big.